src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,276 @@ redirects:
+
+  </Collapser>
+
+  <Collapser
+    id="gap-filling-strategy"
+    title="Gap-filling strategy"
+  >
+  We're creating an alert condition to notify our team of any latency issues with web transaction time. In this case, our application consistently sends New Relic data. There is a constant stream of signals being sent from our application to New Relic and there is no expected gap in signal so we won't need to select a gap-filling strategy. For this use case, and for your first alert, we recommend leaving the **gap-filling strategy** set to **none**. 
+
+  If you have a more inconsistent data set that sends signals to New Relic once every twenty-four hours, then, we'd recommend customizing this feature based on your team's specific needs. 
+
+  Learn more about gap-filling strategies in our [lost signal docs](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/).
+
+  </Collapser>
+</CollapserGroup>
+
+## Connect your condition to a policy [#connect-condition-policy]
+
+  If there are any latency issues with web transaction times, we'd like to receive a notification as soon as possible. The best quick and efficient action is to create an alert condition that will open an incident if web transaction times take too long.
+ 
+  This alert condition is a container that holds all the rules—are we using static or anomaly thresholds, are we using a sliding-window aggregation or just leaving the evaluation period as normal? 
+
+  At this point in the process we now have a fully defined container and we've set all the rules to make sure an incident is opened when we want it to be. Based on the settings above, if our alert condition recognizes this behavior in our system that "violates" the thresholds that we've set, it will create an incident. Now, all we need to do is to attach this container to a policy. 
+  
+  The policy is the sorting system for the incident. When you create a policy, you're creating the tool that organizes all of your incoming incidents. You can connect policies to **[worflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/)** that tell New Relic where you want all this incoming information to go, how often you want it to be sent, and where. 
+
+<CollapserGroup>
+  <Collapser
+    id="select-an-existing-policy"
+    title="Select an existing policy"
+  >
+
+  If you already have a policy that you want to connect your alert condition too, then here's where you would select the existing policy. 
```

Comment:
```suggestion
  If you already have a policy you want to connect to an alert condition, then here's where you would select the existing policy. 
```
@homelessbirds Where's "here"? Is it referring to the docs link mentioned on line 226?